CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 * [FEATURE] New encoding vParquet3 with support for dedicated attribute columns (@mapno, @stoewer) [#2649](https://github.com/grafana/tempo/pull/2649)
 * [ENHANCEMENT] Assert ingestion rate limits as early as possible [#2640](https://github.com/grafana/tempo/pull/2703) (@mghildiy)
 
+* [CHANGE] **Breaking Change** Fix issue where tempo drops the entire batch if one trace triggers an error [#2571](https://github.com/grafana/tempo/pull/2571) (@ie-pham)
```

Comment:
Add details about why this is breaking. I believe it's b/c we are now returning 200/OK when we have partial success on write. A sentence about how this impacts the discarded spans metric would be good too.